src/go/pt-galera-log-explainer/regex/applicative.go

```diff
@@ -92,9 +92,11 @@ var ApplicativeMap = types.RegexMap{
 
 			return logCtx, func(logCtx types.LogCtx) string {
 
-				for _, name := range logCtx.OwnNames {
-					vote, ok := latestConflict.VotePerNode[name]
-					if !ok {
+				for _, localname := range logCtx.OwnNames {
```

Comment:
Minor: I would suggest some refactoring to this chunk
```
				for _, localname := range logCtx.OwnNames {
					if node != localname {
						// don't print other nodes vote, their should be in their own column
						// else it just takes too much place without adding much usability
						continue
					}

					if vote, ok := latestConflict.VotePerNode[localname]; ok {
						return voteResponse(vote, *latestConflict)
					}
				}
```

Reply:
Agree, way better your way